### DISCUSSION TITLE
JUNO中非齐次柏松过程后续概率过小的问题

### DISCUSSION THREAD

**Discussion by Lodfy-Feng** [physics-data/faq#158] [Q&A] [answered]
JUNO中经过计算当达到700ns后平均强度大概是10^-5的量级甚至以下，而强度函数求积分后的均值经过计算1ns都小于1，也就是说这一过程短时间内模拟难以达到10000个光子，请问是我的模拟方法不对吗？

**Comment by Harry-Chen** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
我在描述发光曲线的时候给的是正比，即有一个归一化常数需要自行确定，以保证总光子期望为10000。
